### ISSUE TITLE
nn.FeatureAlphaDropout is missing from the docs

### ISSUE THREAD

**Issue by jbschlosser** [pytorch/pytorch#60563]
## 📚 Documentation

The module form of `nn.FeatureAlphaDropout` is missing from the [torch.nn docs](https://pytorch.org/docs/stable/nn.html) (the [functional form](https://pytorch.org/docs/stable/generated/torch.nn.functional.feature_alpha_dropout.html) is in the docs).

cc @brianjo @mruberry @albanD @jbschlosser

**Comment by sawradip** [comment]
I would like to add this to the docs. 

It would be added under [dropout layers](https://pytorch.org/docs/stable/nn.html#dropout-layers), Right ??

**Comment by jbschlosser** [comment]
@sawradip Sure, go for it. Dropout layers is correct. Should be a one-line change in `docs/source/nn.rst`. Feel free to add me as reviewer.

Might as well tackle #60570 while you're at it- that one goes under Activation layers.

**Comment by sawradip** [comment]
@jbschlosser  I am not totally sure how to add someone as a reviewer, but I have filed a pull request. #60590
